src/scss/core/components/_search.scss

```diff
@@ -103,11 +103,13 @@ $component-name: search;
 
 .header__bottom,
 .#{$component-name}__offcanvas {
-  // stylelint-disable-next-line
+  // stylelint-disable-next-line, selector-id-pattern
   #search_widget {
+    min-width: 20rem;
```

Comment:
Did you test on mobile? I guess it will break on `320px` mobiles because we have a padding